CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Snowpark pandas API Updates
 
 #### New Features
+- Added support for `np.float_power`, `np.mod`, `np.remainder`, `np.greater`, `np.greater_equal`, `np.less`, `np.less_equal`, `np.not_equal`, and `np.equal`.
```

Comment:
maybe let's call this "added numpy compatibility support" to make a difference from the regular Snowpark pandas support